packages/hash/frontend/src/blocks/page/collab/EditorConnection.ts

```diff
@@ -91,6 +93,7 @@ export class EditorConnection {
     switch (action.type) {
       case "loaded": {
         const editorState = createProseMirrorState({
+          accountId: action.accountId,
```

Comment:
This argument isn't necessary – the argument is always equal to `this.accountId`, so just use that – and then you can remove accountId from `EditorConnectionAction` 

```suggestion
          accountId: this.accountId,
```